agent/hcp/telemetry/otel_sink.go

```diff
@@ -77,14 +80,31 @@ func NewOTELSink(opts *OTELSinkOpts) (*OTELSink, error) {
 		return nil, fmt.Errorf("ferror: provide valid context")
 	}
 
+	logger := hclog.FromContext(opts.Ctx).Named("otel_sink")
+
 	// Setup OTEL Metrics SDK to aggregate, convert and export metrics.
-	res := resource.NewSchemaless()
+
+	filterList, err := NewFilterList(opts.Filters)
+	if err != nil {
```

Comment:
Decided not to fail if we can't init some of the filters, in case some are correct, and log an error instead. We should eventually add error metrics to track this, CC @nickethier, let me know if we should fail instead.